tests/test_main.py

```diff
@@ -181,6 +181,38 @@ def test_enrol_bulk_csv(requests_mock, tmp_path):
         assert os.stat("output.csv").st_size != 0
 
 
+def test_unenrol_bulk(moodle_requests_mock, tmp_path):
+    runner = CliRunner()
+    with runner.isolated_filesystem(temp_dir=tmp_path):
+
+        with open('unenrolstudents.csv', 'w') as f:
+            writer = csv.DictWriter(
+                    f,
+                    utils.unenrol_bulk_input_csv_fieldnames()
+                )
+            writer.writeheader()
+            writer.writerows([{utils.CSV_USER_EMAIL: 'tommichaels@gmail.com'}])
+
+        result = runner.invoke(cli, ['unenrol-bulk',
+                                     '2', 'unenrolstudents.csv'],
+                               env=TEST_ENV)
+
+        assert result.exit_code == 0
+
+
+def test_unenrol_bulk_csv(requests_mock, tmp_path):
```

Comment:
Minor: When you remove the use of the fixture below, `requests_mock` can be removed here too.

Reply:
`requests_mock` has been removed.